drivers/ethernet/Kconfig.stm32_hal

```diff
@@ -36,8 +36,9 @@ config ETH_STM32_HAL_RX_THREAD_PRIO
 
 config ETH_STM32_HAL_USE_DTCM_FOR_DMA_BUFFER
 	bool "Use DTCM for DMA buffers"
-	default y
-	depends on SOC_SERIES_STM32F7X
+	default y if SOC_SERIES_STM32F7X
+	default n if SOC_SERIES_STM32H7X
```

Comment:
is this required?

Reply:
Yes, it is. The **Eth. MAC - AHB** is not able to access **DTCM RAM** (see **Table 2** on **p.108** of [reference manual of H745](https://www.st.com/resource/en/reference_manual/dm00176879-stm32h745755-and-stm32h747757-advanced-armbased-32bit-mcus-stmicroelectronics.pdf)). Therefore it must be disabled for H7 series.